toolkit/geo-compose/src/main/java/com/arcgismaps/toolkit/geocompose/MapView.kt

```diff
@@ -81,7 +79,7 @@ import kotlinx.coroutines.launch
  * @param onTwoPointerTap lambda invoked when a user taps two pointers on the composable MapView
  * @param onPan lambda invoked when a user drags a pointer or pointers across composable MapView
  * @param onDrawStatusChanged lambda invoked when the draw status of the composable MapView is changes
- * @param overlay the composable overlays to display on top of the composable MapView. Example, a compass, floorfilter etc.
+ * @param mapViewProxy the [MapViewProxy] to associate with the composable [MapView].
```

Comment:
No need to link to the composable MapView here:
```suggestion
 * @param mapViewProxy the [MapViewProxy] to associate with this composable MapView
```